docs/Orchestration/alternative-schedulers.md

```diff
@@ -1,10 +1,157 @@
 ---
 title: Alternative Schedulers
-draft: true
+id: alternative-schedulers
+description: Support for Alternative Orchestration Solutions
+tags:
+  - jobs
+  - deployment
+  - orchestration
+  - spark-submit
+  - runtime config
 ---
 
-:::caution 🚧 Work in Progress 🚧
+## Basic Spark Submit
 
-TODO
+### Scala Spark Pipelines
+
+Prerequisites:
+
+- (optional) modify ivysettings.xml to point to a custom Maven mirror
+
+Given a Scala Pipeline named "demo_pipeline" with a JAR artifact from [PBT](..%2Fdeployment%2Fprophecy-build-tool%2Fprophecy-build-tool.md)
+called `demo_pipeline-1.0.jar` you could call the following commands to invoke the Main class from the JAR
+file and run the Pipeline on a local Spark cluster.
+
+:::caution
+
+Make sure to use the correct version of `io.prophecy:prophecy-libs_2.12` for your Pipeline.
+Find this version in the `pom.xml` or `pbt_project.yml` in the Pipeline's source code directory.
+Alternatively use a tool like `jdeps` on the jar file itself.
+
+:::
+
+```shell
+spark-submit \
+  --master yarn \
+  --deploy-mode cluster \
+  --driver-memory 8g \
+  --executor-memory 4g \
+  --executor-cores 4  \
+  --packages io.prophecy:prophecy-libs_2.12:3.5.0-8.0.29 \
+  --class io.prophecy.pipelines.demo_pipeline.Main \
+  demo_pipeline-1.0.jar -i default -O "{}"
+```
+
+### PySpark Pipelines
+
+Prerequisites:
+
+- install Python dependencies by installing the WHL file using `pip`
+  - `pip install ./demo_pipeline-1.0-py3-none-any.whl`
```

Comment:
if they have other project/pipeline-level pypi dependencies: those are recorded in the WHL file `install_requires`. 

similarly that is why I am making a change in PBT to have a pom.xml for pyspark projects so that they have an easy wall to gather required jars also.

Reply:
I think this is the easiest way for users to get the python dependencies (including prophecy-libs) . if you have a better idea; please suggest. otherwise let's go with what I have.

Reply:
Sounds good